support/Pipelines/Bbh/Inspiral.py

```diff
@@ -372,6 +372,14 @@ def start_inspiral(
             refinement_level=refinement_level,
             polynomial_order=polynomial_order,
         )
+
+    # Set final time for eccentricity control to 2-3 orbits. This can be set
+    # more dynamically in the future.
+    if eccentricity_control:
+        inspiral_params["FinalTime"] = (
+            500 + 5 * np.pi * inspiral_params["InitialAngularVelocity"]
```

Comment:
Shouldn't this be divided by Omega to get the units right?

Reply:
Ugh yes thanks for catching

Reply:
Thanks! Is there a way to catch this sort of thing? I'm worried we have other latent bugs similar to this one.

Reply:
There are tests for some of these input file parameters but they are mostly "write the same thing in the test that's written in src" so they're not super useful. We really need pipeline tests like #3713 to run a BBH pipeline on CI to catch things like this.

Reply:
Okay, sounds reasonable. That's what I figured. Thanks! :)